README.md

```diff
@@ -108,6 +108,16 @@ It is already possible to build great arg parsing modules on top of what Node.js
 
 It is exceedingly difficult to provide an API which would both be friendly to these Node.js users while being extensible enough for libraries to build upon. We chose to prioritize these use cases because these are currently not well-served by Node.js' API.
 
+----
+
+## Version Matchups
+
+| Node.js | @pkgjs/parseArgs |
+| -- | -- |
+| v18.3.0 | v0.9.1 |
```

Comment:
I think it would be helpful to add a direct link to the config supported on that version

```suggestion
| [v18.3.0](https://nodejs.org/docs/latest-v18.x/api/util.html#utilparseargsconfig) | [v0.9.1](https://github.com/pkgjs/parseargs/tree/v0.9.1#utilparseargsconfig) |
```

Reply:
Good suggestion, thanks.